src/application/components/__tests__/ProjectsList.test.tsx

```diff
@@ -9,7 +9,7 @@ describe('ProjectsList', () => {
   });
 
   it('should render the list of projects', () => {
-    render(<ProjectsList projects={PROJECTS} onProjectClick={noop} />);
+    render(<ProjectsList header="Recent Projects" projects={PROJECTS} onProjectClick={noop} />);
```

Comment:
No need to specify the `header` here and in the next test.
However, it would be great if you could add another test checking that, when passing a `header` prop, the text appears in the document

Reply:
Ok. I will do that.

Reply:
I'm also adding unit tests for the `CreateProjectModal` and `SelectProjectModal`